### ISSUE TITLE
Party Parrot preview breaks if no image URL is given and you adjust image positioning

### ISSUE THREAD

**Issue by fisharu** [parrotify/parrotify.github.io#13]
If you do not enter a URL and adjust the image sliders the preview image breaks. Ideally if there's no valid URL the base image should not change. Thanks a bunch for providing such a fun tool!

**Comment by Omlahid** [comment]
Yes, this is an issue I am aware of, but thanks for reporting it.
I'm glad you're appreciating the service :)
